README.md

```diff
@@ -1,41 +1,158 @@
+| 2080Ti     | Turing      | 11 GB  | GDDR6     | 68  |
+| T4         | Turing      | 16 GB  | GDDR6     | 40  |
+| 3090       | Ampere      | 24 GB  | GDDR6X    | 82  |
+
+**NOTE:** Not implmented yet
+```zsh
+python3 -m pip install habitat
+python3 -c "import habitat"
+```
+
+<h2 id="build">Building from source</h2>
+
+Prerequsites:
+- A system equiped with an Nvidia GPU with properly configured CUDA
+- [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit-archive)
+- [cmake v3.17+](https://github.com/Kitware/CMake/releases)
+- [Habitat pre-trained models](https://zenodo.org/record/4876277)
+
+```zsh
+git clone https://github.com/CentML/habitat.git && cd habitat
+git submodule init && git submodule update
+
+# Download the pre-trained models
+cd analyzer
+curl -O https://zenodo.org/record/4876277/files/habitat-models.tar.gz\?download\=1
+
+# Install the models
+./extract-models.sh
+```
+
+### Building with Docker
+
+1. Run `setup.sh` under `docker/` to build the Habitat container image.
+1. Run `start.sh` to start a new container. By default, your home directory will be mounted inside the container under `~/home`.
+1. Once inside the container, run `install-dev.sh` under `analyzer/` to build and install the Habitat package.
+1. In your scripts, `import habitat` to get access to Habitat. See `experiments/run_experiment.py` for an example showing how to use Habitat.
+
+**Note:** Habitat needs access to your GPU's performance counters, which requires special permissions if you are running with a recent driver (418.43 or later). If you encounter a `CUPTI_ERROR_INSUFFICIENT_PRIVILEGES` error when running Habitat, please follow the instructions [here](https://developer.nvidia.com/ERR_NVGPUCTRPERM) and in [issue #5](https://github.com/geoffxy/habitat/issues/5).
+
+### Building without Docker
+
+```zsh
+# Sanity check, following command should return without errors
+nvidia-smi
+
+# Install CUPTI
+# Find approriate installer for your version of CUDA from:
+# https://developer.nvidia.com/cuda-toolkit-archive
+# For example, the install binary for CUPTI for 11.7.1
+wget https://developer.download.nvidia.com/compute/cuda/11.7.1/local_installers/cuda_11.7.1_515.65.01_linux.run
+
+chmod +x cuda_11.7.1_515.65.01_linux.run
+
+# Make sure to only install the toolkit
+./cuda_11.7.1_515.65.01_linux.run
+
+# Sanity check after installation. Verify following directory exists:
+# On Ubuntu
+ls -la /usr/local/cuda/extras/CUPTI/samples
+
+# On other distributions
+ls -la /opt/cuda/extras/CUPTI/samples
+
+# Install cmake
+wget https://github.com/Kitware/CMake/releases/download/v3.24.0/cmake-3.24.0-linux-x86_64.sh
+chmod +x cmake-3.24.0-linux-x86_64.sh
+mkdir /opt/cmake
+sh cmake-3.24.0-linux-x86_64.sh --prefix=/opt/cmake --skip-license
+ln -s /opt/cmake/bin/cmake /usr/local/bin/cmake
+cmake --version
+
+# Build Habitat
+# In habitat/analyzer run
+./install-dev.sh
+
+# Verify successful build
+python3 -c "import habitat"
```

Comment:
Before importing the user needs to download and extract pretrained models. To do so, run the following:
```bash
wget -q https://www.cs.toronto.edu/~ybgao/saved_models_aug23.tar.gz -O habitat-models.tar.gz
./extract-models.sh habitat-models.tar.gz
```